README.md

```diff
@@ -24,23 +23,23 @@ On the iOS platform, the SDK requires a minimum version of 10.0.
 
 Other Flutter platforms are not currently supported by this SDK.
 
-### Installing the SDK
+### Install the SDK
```

Comment:
Was the "Install SDK" section intentionally left blank? If it's not needed, we can delete header (seems like it would be needed though).